eth/vm/forks/shanghai/computation.py

```diff
@@ -8,12 +8,12 @@
 from eth.exceptions import (
     OutOfGas,
 )
-from eth.vm.forks.paris.computation import ParisComputation
+from eth.vm.forks.paris.computation import ParisMessageComputation
 
 
-class ShanghaiComputation(ParisComputation):
+class ShanghaiComputation(ParisMessageComputation):
```

Comment:
should be `ShanghaiMessageComputation`?

Reply:
This should be outdated as I refactored the whole class across all files based on the last comment.

Reply:
yup, got it